client/src/components/AvatarDisplay/AvatarDisplay.tsx

```diff
@@ -1,10 +1,10 @@
-import Avatar from '@material-ui/core/Avatar';
-import { User } from '../../interface/User';
 import { useState, MouseEvent } from 'react';
+import Avatar from '@material-ui/core/Avatar';
 import Menu from '@material-ui/core/Menu';
 import MenuItem from '@material-ui/core/MenuItem';
```

Comment:
Bundle all together from a `@material-ui/core` import instead

Reply:
Can do. These imports are actually straight from the code on the Material-UI site so it's funny that they didn't do it either.